README.md

```diff
@@ -1,5 +1,16 @@
+# EJ2 DataGrid BlazorServer Globalization with CLDR Files master
```

Comment:
Remove 'EJ2' word while making ReadMe for Blazor Apps.

"Blazor Server DataGrid - Globalization with CLDR Files